plugins/adr/src/components/EntityAdrContent/EntityAdrContent.tsx

```diff
@@ -214,7 +217,7 @@ export const EntityAdrContent = (props: {
 
   return (
     <Content>
-      <ContentHeader title="Architecture Decision Records">
+      <ContentHeader title={t('content_header_title')}>
```

Comment:
hm I'm not a big fan of using free form strings as "keys" for translation messages - I'm wondering if we could establish some kind of pattern where this has better type checks eg. through restricting the type of this param to a set list of enums based off keys of the messages in the translation ref 🤔 

Not something that needs to be addressed in this PR but just thinking out loud and would be curious on your thoughts @Rugvip @mario-mui

Reply:
Well, I'm open to suggestions. I just tried to apply the pattern from the UserSettings example.

Reply:
> hm I'm not a big fan of using free form strings as "keys" for translation messages - I'm wondering if we could establish some kind of pattern where this has better type checks eg. through restricting the type of this param to a set list of enums based off keys of the messages in the translation ref 🤔
> 
> Not something that needs to be addressed in this PR but just thinking out loud and would be curious on your thoughts @Rugvip @mario-mui

<img width="807" alt="image" src="https://github.com/backstage/backstage/assets/7720722/94d81167-a748-4646-83b2-8103a7904648">
<img width="303" alt="image" src="https://github.com/backstage/backstage/assets/7720722/f571a313-694d-4e45-b608-c388f6fd9307">
As I thought, we can get key's prompts by writing comment.